tools/tpp-opt/tpp-opt.cpp

```diff
@@ -33,6 +34,7 @@
 int main(int argc, char **argv) {
   mlir::registerAllPasses();
   mlir::tpp::registerTppCompilerPasses();
+  mlir::tpp::registerTppBundlePasses();
```

Comment:
Shouldn't the bundle pass now replace the tpp-compiler registration?

Reply:
No, these calls only register what's declared in their respective tablegens.
Bundles only contain the bundle passes.